tests/xgboost/test_xgboost_autolog.py

```diff
@@ -554,3 +555,20 @@ def test_xgb_autolog_does_not_break_dmatrix_instantiation_with_data_none():
     """
     mlflow.xgboost.autolog()
     xgb.DMatrix(None)
+
+
+def train(tracking_uri, params):
+    mlflow.set_tracking_uri(tracking_uri)
+    mlflow.xgboost.autolog()
+    dtrain = xgb.DMatrix(*datasets.load_iris(return_X_y=True))
+    with mlflow.start_run() as run:
+        xgb.train(params, dtrain, evals=[(dtrain, "train")])
+    return run.info.run_id
+
+
+def test_xgb_autolog_multi_processing(bst_params, tmpdir):
+    tracking_uri = tmpdir.join("mlruns").strpath
+    with mp.Pool(2) as pool:
+        run_ids = pool.starmap(train, [(tracking_uri, bst_params)] * 2)
+    client = mlflow.tracking.MlflowClient(tracking_uri)
+    assert all(client.get_run(run_id).data.params.keys() >= bst_params.keys() for run_id in run_ids)
```

Comment:
I'm adding tests to reveal autologging integrations that don't work with `multiprocessing`.